ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -134,7 +166,17 @@ private fun MDHeading(
         val text = buildAnnotatedString {
             appendMarkdownChildren(heading, color, allowLinks)
         }
-        MarkdownText(text, color, overriddenStyle, fontWeight, textAlign, allowLinks, modifier)
+        MarkdownText(
+            text,
+            color,
+            overriddenStyle,
+            TextUnit.Unspecified,
```

Comment:
Note that I'm not passing the font size here... This is actually for the headings that we decided not to support for now. In any case, I decided to, instead of hardcode the size to what's given, keep using the one from the app's theme.